lms/static/karma_lms_coffee.conf.js

```diff
@@ -35,6 +35,7 @@ var options = {
         {pattern: 'xmodule_js/common_static/coffee/src/jquery.immediateDescendents.js', included: true},
         {pattern: 'xmodule_js/common_static/js/vendor/jquery-ui.min.js', included: true},
         {pattern: 'xmodule_js/common_static/js/vendor/URI.min.js', included: true},
+        {pattern: 'common/js/vendor/hls.js', included: true},
```

Comment:
I get that we don't need to include hls for lms or cms but why do we need here. ?  i don't see any changes in these tests.